README.md

```diff
@@ -26,6 +26,10 @@ The current Hilla support has some known limitations:
   is not supported
 * Native image compilation does not work
 
+## Tutorial
+
+Get started with `quarkus-hilla` by following the [Quick Start Guide](../../wiki/QuickStart).
```

Comment:
Is the QuickStart ref correct? If I click on it, there comes an issue that it can't be found.

Reply:
Better use a simple branch name when adding links to the wiki next time :)